network-store-server/src/main/resources/iidm.cql

```diff
@@ -483,3 +499,24 @@ SELECT networkUuid, id, voltageLevelId, name, properties, node, p0, q0, r, x, g,
 FROM iidm.danglingLine
 WHERE networkUuid IS NOT NULL AND id IS NOT NULL AND voltageLevelId IS NOT NULL
     PRIMARY KEY (networkUuid, voltageLevelId, id);
+
+CREATE TABLE IF NOT EXISTS iidm.bus (
+    networkUuid uuid,
+    id text,
+    voltageLevelId text,
+    name text,
+    ensureIdUnicity boolean,
```

Comment:
This is not a configured bus datum but just a parameter of bus adder to control if id could be fix automatically.